src/codeu/chat/common/ConversationHeader.java

```diff
@@ -63,4 +63,21 @@ public ConversationHeader(Uuid id, Uuid owner, Time creation, String title) {
     this.title = title;
 
   }
+
+  @Override
+  public boolean equals(Object toCompare)
+  {
+    if(!(toCompare instanceof ConversationHeader))
+      return false;
+    ConversationHeader toCompareConv = (ConversationHeader)(toCompare);
+    if(!this.id.equals(toCompareConv.id))
```

Comment:
I believe so. If it isn't there in the JSON file at all, then the default value is null and it works. In addition, there are some values in Message which I'm setting to null since we just don't have that data and it is working.